dlf/plugins/search/flexform.xml

```diff
@@ -220,6 +220,17 @@
 							</config>
 						</TCEforms>
 					</suggest>
+					<showSingleResult>
+						<TCEforms>
+							<onChange>reload</onChange>
+							<exclude>1</exclude>
+							<label>LLL:EXT:dlf/plugins/search/locallang.xml:tt_content.pi_flexform.showSingleResult</label>
+							<config>
+								<type>check</type>
+								<default>1</default>
```

Comment:
I'd suggest turning this off as default. That is to prevent someone updating the extension from accidently activating a new feature he/she doesn't want. Also, the way the TYPO3 flexform configuration works, the configuration would have to be rewritten to actually change the settings although it would be falsely displayed as activated.

Reply:
I see... then I change the logic "Show single result in listview" to "Show single result in pageview". Then the new feature can be disabled by defult.